spec/metanorma/processor_spec.rb

```diff
@@ -25,53 +22,50 @@
 
   it "generates IsoDoc XML from a blank document" do
     input = <<~"INPUT"
-    #{ASCIIDOC_BLANK_HDR}
+      #{ASCIIDOC_BLANK_HDR}
     INPUT
 
     output = xmlpp(<<~"OUTPUT")
-    #{BLANK_HDR}
-<sections/>
-</bipm-standard>
+      #{BLANK_HDR}
+        <sections/>
+      </bipm-standard>
     OUTPUT
 
     expect(xmlpp(strip_guid(processor.input_to_isodoc(input, nil)))).to be_equivalent_to output
   end
 
   it "generates HTML from IsoDoc XML" do
-    system "rm -f test.xml"
```

Comment:
restore, and add rm of test.html

Reply:
https://github.com/metanorma/metanorma-bipm/pull/115/files#diff-89eebfcbc0f14b6d989517837ca1e94fce4e2ce9a03233641cd936f2b8d2ed94R27 - this will help run tests in isolated tmp directory

Reply:
All file operations should be done in a temp directory.